client/layout/masterbar/style.scss

```diff
@@ -182,7 +182,7 @@ $autobar-height: 20px;
 			top: 9px;
 			left: 50%;
 		width: 8px;
-		z-index: 99999;
+		z-index: z-index( '.masterbar', '.masterbar__notifications-bubble' );
```

Comment:
The notifications bubble is not in the root stacking context, so this rule is optional.